static/app/gettingStartedDocs/rust/rust.tsx

```diff
@@ -22,9 +23,10 @@ export const steps = ({
     configurations: [
       {
         language: 'toml',
+        loading: sourcePackageRegistries?.isLoading,
         code: `
 [dependencies]
-sentry = "0.31.5"
+sentry = "${sourcePackageRegistries?.data?.['sentry.rust'] ?? '0.30.0'}"
```

Comment:
Should this be "0.31.5"?

Reply:
oh yes, I went through this list 2x but missed this one. Thanks 🙏